modules/ob-strategies-with-different-resources.adoc

```diff
@@ -0,0 +1,149 @@
+      image: quay.io/containers/buildah:v1.31.0
+      workingDir: $(params.shp-source-root)
+      securityContext:
+        capabilities:
+          add:
+          - "SETFCAP"
+      command:
+        - /bin/bash
+      args:
+        - -c
+        - |
+          set -euo pipefail
+          # Parse parameters
+        # ...
+        # That's the separator between the shell script and its args
+        - --
+        - --context
+        - $(params.shp-source-context)
+        - --dockerfile
+        - $(build.dockerfile)
+        - --image
+        - $(params.shp-output-image)
+        - --build-args
+        - $(params.build-args[*])
+        - --registries-block
+        - $(params.registries-block[*])
+        - --registries-insecure
+        - $(params.registries-insecure[*])
+        - --registries-search
+        - $(params.registries-search[*])
+      resources:
+        limits:
+          cpu: 250m
+          memory: 65Mi
+        requests:
+          cpu: 250m
+          memory: 65Mi
+  parameters:
+    - name: build-args
+      description: "The values for the args in the Dockerfile. Values must be in the format KEY=VALUE."
+      type: array
+      defaults: []
+    # ...
+----
+
+[id="example-buildah-medium-limit"]
+== Buildah strategy with medium limit
+
+Define the `spec.steps[].resources` field with medium resource limit for the `buildah` strategy, as shown in the following example:
+
+.Example: `buildah` strategy with medium limit
+[source,yaml]
+----
+apiVersion: shipwright.io/v1beta1
+kind: ClusterBuildStrategy
+metadata:
+  name: buildah-medium
+spec:
+  steps:
+    - name: build-and-push
+      image: quay.io/containers/buildah:v1.31.0
+      workingDir: $(params.shp-source-root)
+      securityContext:
+        capabilities:
+          add:
+          - "SETFCAP"
+      command:
+        - /bin/bash
+      args:
+        - -c
+        - |
+          set -euo pipefail
+          # Parse parameters
+        # ...
+        # That's the separator between the shell script and its args
+        - --
+        - --context
+        - $(params.shp-source-context)
+        - --dockerfile
+        - $(build.dockerfile)
+        - --image
+        - $(params.shp-output-image)
+        - --build-args
+        - $(params.build-args[*])
+        - --registries-block
+        - $(params.registries-block[*])
+        - --registries-insecure
+        - $(params.registries-insecure[*])
+        - --registries-search
+        - $(params.registries-search[*])
+      resources:
+        limits:
+          cpu: 500m
+          memory: 1Gi
+        requests:
+          cpu: 500m
+          memory: 1Gi
+  parameters:
+    - name: build-args
+      description: "The values for the args in the Dockerfile. Values must be in the format KEY=VALUE."
+      type: array
+      defaults: []
+    # ...
+----
+
+When the resource definition is configured for a strategy, you must reference the strategy in your `Build` CR, as shown in the following example:
```

Comment:
"is configured for" or "includes"?

Reply:
We cannot use "includes" because a resource definition does not include a startegy, it is actually the other way round (a strategy includes a resource definition). I have tweaked the sentence a bit for clarity:
After configuring the resource definition for a strategy, you must reference the strategy in your `Build` CR